canonical/juju-dashboard
# WD-8431 - Fetch and store secrets

## Done

- Fetch and store secrets.

## QA

- Set up a controller with Juju 3.3.1.
- Add some secrets to a model with: `juju add-secret my-apitoken token=34ae35facd4`
- Open the dashboard and navigate to the model.
- Click on the Secrets tab.
- The secrets should get fetch and displayed in a list.

## Details

https://warthogs.atlassian.net/browse/WD-8431
